modules/tutor/src/test/GlickoTest.scala

```diff
@@ -2,15 +2,15 @@ package lila.tutor
 
 class GlickoTest extends munit.FunSuite:
 
-  test("glicko for arbitrary outcomes") {
+  test("glicko for binary outcomes") {
     assertEquals(
-      TutorGlicko.scoresRating(
+      TutorGlicko.outcomesRating(
         lila.rating.Perf.default,
         List(
-          (1400, 0.8f),
-          (1700, 0.6f)
+          (1400, true),
```

Comment:
true is not 0.8f. We were testing for intermediary results here.

Reply:
I restored this test and added a comment with `FloatingResult` that we aren't asserting that the result is within unit interval [0.0d, 1.0d].